lib/merge_ats_client.rb

```diff
@@ -23,17 +23,18 @@
 require 'merge_ats_client/models/activity'
 require 'merge_ats_client/models/activity_type_enum'
 require 'merge_ats_client/models/application'
+require 'merge_ats_client/models/application_request'
 require 'merge_ats_client/models/attachment'
 require 'merge_ats_client/models/available_actions'
 require 'merge_ats_client/models/candidate'
-require 'merge_ats_client/models/data_passthrough'
```

Comment:
we want to keep these for now:
require 'merge_ats_client/models/data_passthrough'
require 'merge_ats_client/models/end_user_details'